modules/appeals_api/app/controllers/appeals_api/notice_of_disagreements/v0/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -1,12 +1,64 @@
 # frozen_string_literal: true
 
 module AppealsApi::NoticeOfDisagreements::V0::NoticeOfDisagreements
-  # rubocop:disable Layout/LineLength
-  class EvidenceSubmissionsController < AppealsApi::V2::DecisionReviews::NoticeOfDisagreements::EvidenceSubmissionsController
-    # rubocop:enable Layout/LineLength
+  class EvidenceSubmissionsController < AppealsApi::ApplicationController
```

Comment:
Similar to the changes I made for supplemental claims evidence submission, I was able to break the inheritance relationship here